common-develocity-gradle-configuration-kotlin/settings.gradle.kts

```diff
@@ -21,18 +21,18 @@ buildCache {
         isEnabled = true
     }
 
-    // Use the Gradle Enterprise connector's access key based authentication.
-    // This is available in Gradle Enterprise 2022.3+ and Gradle Enterprise Plugin 3.11+.
+    // Use the Develocity connector's access key based authentication.
+    // This is available in Develocity 2022.3+ and Develocity Plugin 3.11+.
     remote(gradleEnterprise.buildCache) {
         isEnabled = true
         isPush = isCI
     }
 
     // Use Gradle's built-in access credentials.
-    // This is available in all Gradle Enterprise and Gradle Enterprise Plugin versions.
+    // This is available in all Develocity and Develocity Plugin versions.
     /**
     remote(HttpBuildCache::class) {
-        url = uri("https://enterprise-samples.gradle.com/cache/") // adjust to your GE server, and note the trailing slash
+        url = uri("https://develocity-samples.gradle.com/cache/") // adjust to your Develocity server, and note the trailing slash
         isAllowUntrustedServer = false // ensure a trusted certificate is configured
         credentials {
             username = System.getenv("GRADLE_ENTERPRISE_CACHE_USERNAME")
```

Comment:
Shoudln't we rename to `DEVELOCITY_CACHE_USERNAME` / `DEVELOCITY_CACHE_PASSWORD` ?